src/components/lightning/Lightning.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const Lightning = () => {
+  return (
+    <img
+      src="/img/whoLikesMe/who_liked_me_red.svg"
+      alt="Lightning"
+      style={{
```

Comment:
plz move style to according hook

Reply:
moved style to according hook in Lightning component